stm32-modules/flex-stacker/firmware/tof_sensor/tmf8820_image.c

```diff
@@ -0,0 +1,227 @@
+/* http://srecord.sourceforge.net/ */

```

Comment:
this will work, but I would love if we could do something like the following:
- Have the image in the repo as a binary, straight up
- do one of the following:
  - generate a source file like this from the binary image using a python script or using `xxd` or something, integrated as a cmake task, in a similar way to how we handle versions
  - pack it into the linked binary directly